--- a/debian/rules	2025-10-16 23:49:36.154702328 +0200
+++ b/debian/rules	2025-10-16 23:49:36.156702328 +0200
@@ -29,3 +29,8 @@
 override_dh_auto_configure:
-	dh_auto_configure --  -DUSE_SYSTEM_LIBINDI=1 -DUSE_SYSTEM_GTEST=1 -DUSE_SYSTEM_LIBUSB=1 -DOPENSOURCE_ONLY=1
+	dh_auto_configure --  -DUSE_SYSTEM_LIBINDI=1 -DUSE_SYSTEM_GTEST=1 -DUSE_SYSTEM_LIBUSB=1 -DOPENSOURCE_ONLY=0
 
+
+# Tell dh_shlibdeps where to find proprietary camera libraries
+# These are bundled binary-only libraries in /usr/lib/phd2/
+override_dh_shlibdeps:
+	dh_shlibdeps -l$(CURDIR)/debian/phd2/usr/lib/phd2
